### PR TITLE
docs: add "Link a Server to an Application" post

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Add post "Link a Server to an Application", guiding the user to link the application to a server before deploying.

## Issues resolved by this PR

- [x] [DVN-10980]
- [x] [DVN-10780]
- [x] [DVN-10828]

## More info

Post Preview
image

[DVN-10980]: https://devopness.atlassian.net/browse/DVN-10980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DVN-10780]: https://devopness.atlassian.net/browse/DVN-10780?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10828]: https://devopness.atlassian.net/browse/DVN-10828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/applications/link-server-to-application.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to link a server
+1. On the upper-right corner of the application details view, click `...`
+1. Use the drop-down menu to choose `Linked resources`
+1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
+1. Follow the prompts then click `Link`
+    - The selected server will be listed in the `DEPENDS ON` table on the `Linked resources` list
```
Comment (jfoliveira) [inline docs/applications/link-server-to-application.md:24]: `Follow the prompts ...` + `the selected ...` is really the pattern we've been following in other topics that use select/drop down elements? 🤔 
`Follow the prompts` was usually adopted (by you) when there were more than one thing/element to interact with.

Please re-consider which approach is the one you recommend and let's stick to a single one.

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/applications/link-server-to-application.md (hunk 1 above)
Comment (thlmenezes) [reply docs/applications/link-server-to-application.md]: I agree, it's a pattern change.

Added a new step to the list, could you please re-review?

Comment (thlmenezes) [reply docs/applications/link-server-to-application.md]: Applied the same pattern used in `docs/cloud-provider-credentials/add-aws-credential.md`

```md
...
1. Follow the prompts then click `Create user`
1. In the `Users` list, click the username link of the user recently created
...
```

File: docs/applications/link-server-to-application.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to link a server
+1. On the upper-right corner of the application details view, click `...`
+1. Use the drop-down menu to choose `Linked resources`
+1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
+1. Follow the prompts then click `Link`
+1. In the `Linked Resource` list, the selected server will be listed in the `DEPENDS ON` table
```
Comment (thlmenezes) [inline docs/applications/link-server-to-application.md:24]:
```suggestion
1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table
```

File: docs/applications/link-server-to-application.md (hunk 1 above)
Comment (thlmenezes) [reply docs/applications/link-server-to-application.md]: @jfoliveira could you review if the applied changes solves the thread?

File: docs/applications/link-server-to-application.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to link a server
+1. On the upper-right corner of the application details view, click `...`
+1. Use the drop-down menu to choose `Linked resources`
+1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
+1. Follow the prompts then click `Link`
+1. In the `Linked Resource` list, find the selected server from the previous step in the `DEPENDS ON` table
```
Comment (jfoliveira) [inline docs/applications/link-server-to-application.md:24]: `find the selected server`? Why? For what? 

If the intention is to have a closing statement, it should not be a "step/order" to the reader, should just be an informational note
```
> In the <element>, the <info> can be seen in <info location>
> In the <element>, the <info> will be displayed in <info location>
etc
```

If it's an imperative instruction, it needs to guide the user o do something with the info that was found
```
find <info> to <...>
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/applications/link-server-to-application.md (hunk 3 above)
Comment (thlmenezes) [inline docs/applications/link-server-to-application.md:24]:
```suggestion
1. In the `Linked Resource` list, the recently linked server can be seen in the `DEPENDS ON` table
```

Comment (thlmenezes) [reply docs/applications/link-server-to-application.md]: Post updated

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**